### DISCUSSION TITLE
Use H801 to drive up to 5 independent led strips (eg. 5 different white strips)

### DISCUSSION THREAD

**Discussion by FulvioSpelta** [arendst/Tasmota#10344] [Q&A] [answered]
Hi all, I'd like to use H801 to drive different lighting zones in the house so I'd like to use every different "line" to drive a white lighting strip.

Is it possible through configuration or using a custom module ?
I'm avilable to compile myself but i'm not a programmer so every hint is needed and welcome.

To be completed I'd like to use pin to drive a relay that to switch the ac-dc power supply avoiding waste of power in off mode.

**Complete idea:**
mini power supply to have H801 alway on
H801 drive relay to  switch power for strips as needed
H801 that drive brightness for up to 5 strips

Every suggestion even for different hw are welcome.

Thanks

**Comment by barbudor** [comment, marked as answer]
Hi @FulvioSpelta
Happy new year

H801 is probably the best fit for your project apart for a dedicated DIY board. There are 5 PWM channels. 
For Tasmota to use each PWM independently you need to enter configuration command [`Setoption68 1 1`](https://tasmota.github.io/docs/Commands/#setoption68)
However if you need an integration into an home automation you will probably need manual setup as generally Tasmota integration is to appear as a single light. 

Regarding adding 5 relays this may be the more challenging part. Although the template shows enough available pins, they may be hard to reach. A simpler solution would be to use only 2 pins as i2c with an IO expander like PCF8574 or MCP23017.

I see this easily achievable without any compilation or coding and a little bit of soldering. 

Cheers

**Comment by FulvioSpelta** [comment]
Thank you very much for immediate reply.
About the relais, I'd like to use only 1 relais because the AC-DC power supply will be unique for all the 5 strips; obviously it should switch when at least 1 strip is ON.

About integration, I'd like to integrate in openhab using a custom mqtt configuration to see every strip as single dimmerable light

Thanks again
f

**Reply by sfromis** [reply]
I'd say first monitor power usage of idle H801 before deciding if you need a relay. My experience with idle H801 and a small 12V power supply is near 0.6W, about double of a D1 mini with a USB power supply. Both measured from mains, including loss in the power supply, where I suspect the 12V one to burn more heat than the 5V one.

Simplest relay would be to have a relay for power to the H801, but that would take first turning the relay on, and wait a few seconds till the H801 has booted. Or you could analyze the H801 board, and modify it if you find a good spot for switching the led driver side. As long as you use a custom config, it should work smoothly with SetOption68 for split channels.

**Reply by FulvioSpelta** [reply]
Thanks, I've just ordered the mini-ps I'll build a prototype then i'll post consumption info.

**Comment by FulvioSpelta** [comment]
First test using a laboratory power supply so measuring the consumption of the H801 itself only.
Steps
1. supply only the H801 without any led strip connected
2. wait few minutes for wifi connetion
3. all strip in off conditions (avoiding the green led on too)
4. sleepmode dynamic as default

It seems that there no difference between 5V and 12V:
![image](https://user-images.githubusercontent.com/1832999/103469617-708a1580-4d67-11eb-931d-4e2d0358e241.png)
![image](https://user-images.githubusercontent.com/1832999/103469633-9f07f080-4d67-11eb-8591-f3db71602d53.png)
